### ISSUE TITLE
Optional counter

### ISSUE THREAD

**Issue by klpx** [outworkers/phantom#697]
Counter column in cassandra by default contains nothing (`null` for selecting).
So if value is not initialized i've got:
```ERROR MyStorage$myCounters: Unable to parse value for column my_counter from row
com.outworkers.phantom.builder.primitives.Primitive$$anon$2: Column my_counter is null```

Example: i use it just like https://github.com/outworkers/phantom/wiki/Counter-columns

**Comment by alexflav23** [comment]
Hi @klpx Just want to confirm which version of phantom you are using.

**Comment by klpx** [comment]
Hi! I use 2.9.1

**Comment by rtatavarty** [comment]
I am using 2.12.1, facing the same issue. Any workaround (like substituting OptionalLongColumn) available for this?

**Comment by alexflav23** [comment]
Hi @klpx @rtatavarty ,

We are patching this up tomorrow, apologies for the delay!

Regards,

**Comment by rtatavarty** [comment]
Thanks @alexflav23 for the info!

**Comment by alexflav23** [comment]
@rtatavarty @klpx Fixed in 2.13.1, it will be live today on Maven Central.
